internal/api/starlark_test.go

```diff
@@ -3,100 +3,138 @@ package api
 import (
```

Comment:
great test, but what's it doing in the api package?

Reply:
Where else would you put it? Being in starlark creates a circular dependency because it's set up like an API test.